guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -40,6 +40,13 @@ The following updates describe the latest improvements to the `{{site.data.var.c
 
 #### Resolved Issues
 
+-  <!--MAGECLOUD-2736-->Resolved an issue preventing a successful installation of Magento when specifying a custom database configuration.
+
+-  <!--MAGECLOUD-2740-->Fixed an issue that neglected to include each website locale in the `systems` section of the `config.local.php` file after running the `config:dump` command.
+
+-  <!--MAGECLOUD-2797-->Fixed an issue causing warm-up errors during the post-deploy phase by correcting the source base URL reference.
+
+-  <!--MAGECLOUD-2850-->Fixed an issue with the `setup:di:compile` process for Amazon payment module.
```

Comment:
This seems like it needs more info.  I read some of the tickets-- the issue was no read/write permissions on the generated directory after Cloud deployment.

Reply:
I have read the tickets and am awaiting development feedback on this.